components/todo-item.js

```diff
@@ -1,11 +1,63 @@
-import React from "react";
+import React, { useState } from 'react';
+
 import styled from "styled-components";
 
-export default function TodoItem() {
-  return <Wrapper>Todo</Wrapper>;
+export default function TodoItem({ id, content, date, todoRemove }) {
+  return (
+    <Wrapper>
+      <TodoItemBox>
+        <Todo>{content}</Todo>
+        <Day>{date}
+        <ClearBtn onClick={(() => {todoRemove(id)})}>완료</ClearBtn>
+        </Day>
+      </TodoItemBox>
+    </Wrapper>
+    );
 }
```

Comment:
```suggestion
export default function TodoItem({ todo, onDelete }) {
  return (
    <Wrapper>
      <Row>
        <Todo>{content}</Todo>
        <Date>{date}</Date>
      </Row>
       <DeleteButton onClick={onDelete}>완료</DeleteButton>
    </Wrapper>
    );
}
```
1. props가 바뀐 건 ```components/todo-list.js``` 참고해주세요!
2. 네이밍 바뀐 건 아래 styled-components에 달린 리뷰들 참고해주세요
3. ```TodoList```가 ```TodoItem```의 index까지 포함해서 onDelete를 넘겼기 때문에 여기에서는 onClick에 onDelete 함수만 넣어주면 잘 작동하게 됩니다.
4. 결과 화면의 TodoItem의 구조를 나눠보면 Date 텍스트를 나타내는 태그 Date가 DeleteButton을 자식으로 가지는 게 어색합니다.
구조를 잘 나눠보면 Todo내용, 날짜가 한 행에 있고 그 아래 행 맨 오른쪽에 완료 버튼이 있기 때문에 코드를 위와 같이 작성해주시는 게 좋습니다.
DeleteButton의 style은 아래 리뷰를 참고해주세요.